src/core/text/Text.js

```diff
@@ -273,7 +273,10 @@ Text.prototype.updateText = function (respectDirty)
     if (navigator.isCocoonJS)
```

Comment:
Ha, yeah true. We are basically banking on the width/height assignments clearing the canvas without this clear here. But, like you found here that doesn't always work. I wonder if it might be a case of preferring "correctness" over bleeding perf. I don't particularly feel strongly either way.